llvm/lib/Target/AArch64/AArch64MachineFunctionInfo.cpp

```diff
@@ -82,6 +83,25 @@ static bool ShouldSignWithBKey(const Function &F, const AArch64Subtarget &STI) {
   return Key == "b_key";
 }
 
+static bool hasELFSignedGOTHelper(const Function &F,
```

Comment:
Please add a brief doc comment

Reply:
Added, see 76ca5c45a02d2d35ab54ae562cdab0f70ecd14e3, thanks for suggestion